__tests__/app.test.js

```diff
@@ -229,3 +229,52 @@ describe('7-post', () => {
     .expect(404)
   })
 })
+
+describe('8-patch', () => {
+  test('status 200 with positive number', () => {
+    const voteInc = {voteInc: 5}
+    return request(app)
+    .patch('/api/reviews/3')
+    .send(voteInc)
+    .expect(200)
+    .then(({body}) => {
+      expect(body.review).toEqual(
+        expect.objectContaining({
+          votes: 10,
+          category:expect.any(String),
+          created_at:expect.any(String),
+          review_id:expect.any(Number),
+          designer:expect.any(String),
+          owner:expect.any(String),
+          review_id: 3,
+          review_img_url:expect.any(String),
+          title:expect.any(String),
+          review_body:expect.any(String)
+        })
```

Comment:
For this test I would amend the test to check the actual contents of review number 3, instead of just checking data types... I can see you're checking the votes property has changed to 10, but I would change all the other properties to show the specific review values.